Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/CriticalLogsAnalyticsHandler.cs

```diff
@@ -30,38 +33,40 @@ private void TrackUnhandledException(object sender, UnhandledExceptionEventArgs
                 { "type", "unhandled exception" },
                 { "category", IAnalyticsController.UNDEFINED },
                 { "scene_hash", IAnalyticsController.UNDEFINED },
-                { "message", e.Message },
+                { "message", IsPayloadSizeValid(e.Message) ? e.Message : "message is too long" },
             });
         }
 
-        public void Log(LogType logType, ReportData reportData, Object context, object message)
+        public void Log(LogType logType, ReportData reportData, Object context, object messageObj)
```

Comment:
Just a thought, would it make sense to change `messageObj` from `object` to `string`? What do you thinkg?

Reply:
Not sure. This is how `IReportHandler` was designed by @mikhail-dcl , so probably there are reasons to be it an object. I am just implementing the interface here...

Reply:
Oh I see, thanks! I just love highly typed signatures :P